### DISCUSSION TITLE
我在steam下载一个游戏，他好像走了代理在下载

### DISCUSSION THREAD

**Discussion by hao-888** [kkkgo/PaoPaoGateWay#71] [问和答]
我应该怎么不让他走代理，但是不是下载的功能，要走

**Comment by hao-888** [comment]
![image](https://github.com/kkkgo/PaoPaoGateWay/assets/64783863/caa82aa7-23fe-43f9-a5bf-8db4a1183256)
epic也是这样

**Comment by kkkgo** [comment]
参考 https://github.com/kkkgo/PaoPaoDNS/discussions/47
另外steam可以在设置里面选择中国下载节点。
